### ISSUE TITLE
HTTP storage endpoints forced to HTTPS

### ISSUE THREAD

**Issue by LachlanStuart** [lithops-cloud/lithops#708]
In addition to the compute side, we've been using Lithops's Storage API directly on the host computer instead of calling boto/ibm-cos-sdk, as it allows us to easily switch between AWS S3, IBM COS and local storage. 

I ran into one issue: for our development computers we use [MinIO](https://hub.docker.com/r/minio/minio/) as a local S3-compatible server so that we don't need a fast internet connection for local development. As MinIO runs on a local container ( http://storage:9000 ), HTTPS isn't an option. However, Lithops [overwrites the configured endpoint to force it to HTTPS](https://github.com/lithops-cloud/lithops/blob/master/lithops/storage/backends/aws_s3/aws_s3.py#L29) ([also in the COS backend](https://github.com/lithops-cloud/lithops/blob/master/lithops/storage/backends/ibm_cos/ibm_cos.py#L51)), causing the connection to fail.

This storage is also directly accessed from the browser (uploading datasets & downloading images), so we can't just switch to the localhost backend for these buckets. I realize this kind of scenario doesn't quite fit into the core use-case of Lithops, but the Storage API is really nice and I'd love to be able to use it throughout our application so that our storage code is cloud-agnostic.

My suggested fix would be to remove `.replace('http:', 'https:')` and add a warning instead, e.g.:
```python
if 'http:' in service_endpoint:
    logger.warning('Endpoint {} is insecure - it is recommended to change this to https://'.format(service_endpoint))
```

**Comment by JosepSampe** [comment]
Hi @LachlanStuart, as the interface is exactly the same in `aws_s3`, `ibm_cos` and `ceph`, if you prefer I can add a new storage backend called `minio` and add a new configuration section in config for it. This way you don't need to configure local `minio` backend as `aws_s3`, and you can differentiate between them in config.

**Comment by LachlanStuart** [comment]
Hi @JosepSampe That's up to you - it sounds like substantially more work than my suggestion, but it would also solve it. 

Our existing code doesn't differentiate between S3/MinIO - we just use different values for the S3 config options. For this storage type we use S3 for production and MinIO only for local development. (And for serverless compute we use COS for prod and COS or localhost for dev).  I don't foresee any scenario where we'll need both MinIO and S3 in the same environment.

**Comment by JosepSampe** [comment]
@LachlanStuart I finally added both changes: I added the warning you stated above in `aws_s3`, and I also added a new `MinIO` Storage backend, so you can now use the one you prefer.

**Comment by LachlanStuart** [comment]
@JosepSampe Awesome! Thank you very much

**Comment by LachlanStuart** [comment]
Fixed in #709
